integration-test/src/main/resources/application.yml

```diff
@@ -88,6 +88,7 @@ integrationtest:
     targetRuntimeVersion: 7.2.13
     distroXUpgradeCurrentVersion: 7.2.7
     distroXUpgradeTargetVersion: 7.2.13
+    imageCatalogUrl3rdParty: https://cloudbreak-imagecatalog.s3.amazonaws.com/v3-prod-cb-image-catalog.json
```

Comment:
We should introduce this new variable to the [cloudbreak-ansible-playbooks/jenkins/build-scripts/cloudbreak-e2e-tests.sh](https://github.com/hortonworks/cloudbreak-ansible-playbooks/blob/master/jenkins/build-scripts/cloudbreak-e2e-tests.sh) and the related Jenkins jobs (longrunning ones and the PR AWS ones).

Reply:
Based on our conversation, we won't make external parameters for these actually. So this comment is out of the scope right now.